src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -177,7 +177,6 @@ public SyncTree BuildSyncTree(DiskState currentState, DiskState previousTree, IE
                 return file;
             })
             .Where(f => !f.TryGetAsDeletedFile(out _))
-            .Where(f => !IsIgnoredPath(f.TargetPath))
```

Comment:
As discussed on Slack, there is prob a tweak that can be done so that we don't need this